lib/project_types/theme/tasks/ensure_themekit_installed.rb

```diff
@@ -20,19 +19,20 @@ def call(ctx)
         begin
           releases = JSON.parse(Net::HTTP.get(URI(URL)))
           release = releases["platforms"].find { |r| r["name"] == OSMAP[ctx.os] }
-          ctx.puts(ctx.message('ensure_themekit_installed.downloading', releases['version']))
-          File.write(FILENAME, Net::HTTP.get(URI.parse(release["url"])))
-          ctx.puts(ctx.message('ensure_themekit_installed.verifying'))
+          ctx.puts(ctx.message('theme.ensure_themekit_installed.downloading', releases['version']))
+          File.write(Themekit::THEMEKIT, Net::HTTP.get(URI.parse(release["url"])).force_encoding("UTF-8"))
```

Comment:
Is this because https://shopify-themekit.s3.amazonaws.com/releases/latest.json isn't UTF-8?

Reply:
This ended up being an issue with `Net::HTTP`, which I did not previously realize. This band-aid fix will be changed.